src/components/control-bar/SeekBar.js

```diff
@@ -108,14 +133,21 @@ export default class SeekBar extends Component {
         getPercent={this.getPercent}
         stepForward={this.stepForward}
         stepBack={this.stepBack}
+        childrenToMerge={this.getDefaultChildren()}
       >
-        <LoadProgressBar
-          buffered={buffered}
-          currentTime={time}
-          duration={duration}
-        />
-        <MouseTimeDisplay duration={duration} mouseTime={mouseTime} />
-        <PlayProgressBar currentTime={time} duration={duration} />
+        {renderGrandchildren ? (
+          children.props.children
+        ) : (
+          <>
```

Comment:
Non-blocking feedback. I believe this fragment is not necessary, but am I missing something?

Reply:
I get an error if I don't have it 🙃

Reply:
Oh duh. Yeah I missed the part about it being the "else" of a ternary expression.